docs/book/features/file_volume.md

```diff
@@ -0,0 +1,18 @@
+# File volumes
+
+The vSphere Vanilla CSI driver version 2.0 and above supports file volumes backed by vSAN File shares using NFSv4 protocol to be statically/dynamically created and mounted by stateful containerized applications. This feature allows you to reference the same shared data among multiple pods spread across different clusters using the `ReadWriteMany` and `ReadOnlyMany` access modes making it an absolute necessity for applications that need shareability.
+
+Before you setup file services in your environment keep in mind that if you have file shares being shared across more than one clusters in your vCenter, deleting a PVC with reclaim policy set to `Delete` in any one cluster may delete the underlying file share causing the volume to be unavailable for the rest of the clusters.
+
+Also note that the file volumes feature doesn't work in conjunction with the topology aware zones, extend volume and encryption features.
+
+Proceed to the requirements section below to enable this feature in your environment. After you setup file services, to get started with vSphere CSI file services integration with your applications, check this [video](https://youtu.be/GUtG-4urGFA). We have also provided few [Try-out YAMLs](https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/example/vanilla-k8s-file-driver) for Storage Class, PVC, Pods for your convenience.
+
+## Requirements
+
+Check the CSI, vSphere compatibility matrices and the supported features section to verify if your environment conforms to all the required versions. Also check the limitations section to understand if this feature can cater to your needs.
```

Comment:
link to respective pages for compatibility matrices and supported features section would be useful.